### PR TITLE
[benchmark] Fix swiftpm based benchmark build on Linux.

### PR DESCRIPTION
**Pull request by gottesmm**

### CODE REVIEW

**Comment by gottesmm** [comment]
Please test with following pull request:
apple/swift-integration-tests#36

@swift-ci please test linux platform

**Comment by gottesmm** [comment]
Please test with following pull request:
apple/swift-integration-tests#36

@swift-ci please test linux platform

**Comment by gottesmm** [comment]
Please test with following pull request:
apple/swift-integration-tests#36

@swift-ci please test linux platform

**Comment by gottesmm** [comment]
Please test with following pull request:
apple/swift-integration-tests#36

@swift-ci please test linux platform

**Comment by gottesmm** [comment]
Please test with following pull request:
apple/swift-integration-tests#36

@swift-ci please test linux platform

**Comment by gottesmm** [comment]
Please test with following pull request:
apple/swift-integration-tests#36

@swift-ci please test linux platform

**Comment by gottesmm** [comment]
Please test with following pull request:
apple/swift-integration-tests#36

@swift-ci please test linux platform

**Comment by gottesmm** [comment]
Please test with following pull request:
apple/swift-integration-tests#36

@swift-ci please test linux platform

File: benchmark/Package.swift [hunk 1]
```diff
@@ -3,13 +3,19 @@
 import PackageDescription
 import Foundation
 
+var unsupportedTests: Set<String> = ["ObjectiveCNoBridgingStubs"]
+#if !os(macOS) && !os(iOS) && !os(watchOS) && !os(tvOS)
```
Comment (aciidb0mb3r) [inline benchmark/Package.swift]: It'll be better to use `canImport(Darwin)` or `canImport(ObjectiveC)`

**Comment by gottesmm** [comment]
@swift-ci build-toolchain

**Comment by gottesmm** [comment]
@swift-ci build toolchain

**Comment by gottesmm** [comment]
Please test with following pull request:
apple/swift-integration-tests#36

@swift-ci please test linux platform

**Comment by gottesmm** [comment]
@swift-ci smoke benchmark

**Comment by gottesmm** [comment]
Please test with following pull request:
apple/swift-integration-tests#36

@swift-ci please test linux platform

**Comment by gottesmm** [comment]
@swift-ci smoke benchmark

**Comment by gottesmm** [comment]
Please test with following pull request:
apple/swift-integration-tests#36

@swift-ci please test linux platform

**Comment by gottesmm** [comment]
@swift-ci smoke test os x platform

**Comment by palimondo** [comment]
Looking at those two compiler errors @aciidb0mb3r [points out above](https://github.com/apple/swift/pull/20121#issuecomment-434025748)… Aren't these bugs in how `Glibc` imports the `RUSAGE_SELF` and `_SC_PAGESIZE ` constants?

**Comment by gottesmm** [comment]
Feel free to file on swift.org.

**Comment by palimondo** [comment]
Filed [SR-9152](https://bugs.swift.org/browse/SR-9152).

**Pull request merged**